python/tank/commands/setup_project_params.py

```diff
@@ -308,6 +307,13 @@ def get_configuration_readme(self):
 
         return self._config_template.get_readme_content()
 
+    def get_default_storage_name(self):
```

Comment:
can this be a property?

Reply:
I'll change it. There aren't any properties in this code and I was following the existing style, but makes more sense that way.